README.md

```diff
@@ -4,18 +4,15 @@ Kubernetes is now available as a DC/OS package to quickly, and reliably run Kube
 
 ![](docs/assets/ui-install.gif)
 
-**NOTE:** The latest `dcos-kubernetes-quickstart` doesn't support any Kubernetes framework version before
-`1.2.0-1.10.5` due the changes how the Kubernetes API is exposed.
+**NOTE:** The latest `dcos-kubernetes-quickstart` doesn't support any Kubernetes framework version before `2.0.0-1.12.1`. The reason is that now creating Kubernetes clusters requires installation of the Kubernetes Manager, package: `Kubernetes`.
```

Comment:
I'd write `requires the installation of the Mesosphere Kubernetes Engine (link to the docs).`